plugin-server/src/cdp/cdp-api.ts

```diff
@@ -34,6 +37,7 @@ export class CdpApi {
     private hogTransformer: HogTransformerService
     private hogFunctionMonitoringService: HogFunctionMonitoringService
     private cdpSourceWebhooksConsumer: CdpSourceWebhooksConsumer
+    private MAILJET_SECRET = process.env.MAILJET_SECRET || ''
```

Comment:
Please never use `process.env` - always go via the dedicated `PluginServerConfig/Hub`